Flutter/DateRangePicker/getting-started.md

```diff
@@ -58,16 +60,23 @@ import 'package:syncfusion_flutter_datepicker/datepicker.dart';
 
 After importing the package, initialize the date range picker widget as a child of any widget. Here, the date range picker widget is added as a child of the scaffold widget.
 
+* [initialDisplayDate](https://pub.dev/documentation/syncfusion_flutter_datepicker/latest/datepicker/SfDateRangePicker/initialDisplayDate.html) - Sets the initially displayed date. Default is today's date `DateTime.now()`.
```

Comment:
Why did you added these contents here? This is getting started page.

The explained properties are not used in the following codes